database/src/seeds/03_basic_project_survey_setup.ts

```diff
@@ -346,14 +346,15 @@ const insertSurveyLocationData = (surveyId: number) => `
  * SQL to insert Survey data
  *
  */
-const insertSurveyData = (projectId: number, surveyName?: string) => `
+const insertSurveyData = (projectId: number, surveyProgress: number, surveyName?: string) => `
```

Comment:
Nit: probably don't need this as a parameter. Since we're always passing `1` as the survey progress, I would just hardcode this value in the actual insert query (Line 367)

Reply:
might also be worth giving a valid range of values so the seeded surveys can be `in progress` or `completed` as well

Reply:
Roger. I added ```${Math.ceil(Math.random() * 3)}``` in line 367, where valid values are 1, 2, and 3.